pkg/kubectl/describe.go

```diff
@@ -2385,6 +2387,36 @@ func (s *StorageClassDescriber) Describe(namespace, name string, describerSettin
 	})
 }
 
+type PodDisruptionBudgetDescriber struct {
+	clientset.Interface
+}
+
+func (p *PodDisruptionBudgetDescriber) Describe(namespace, name string, describerSettings DescriberSettings) (string, error) {
```

Comment:
I'll wait for the other PR to land and will update this accordingly, in that case.